src/libraries/System.Private.Xml/tests/Misc/System.Xml.Misc.Tests.csproj

```diff
@@ -2,8 +2,11 @@
   <PropertyGroup>
     <RootNamespace>XmlMiscTests</RootNamespace>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <!-- ActiveIssue XUnit crashes on Apple Silicon
```

Comment:
I don't have a strong preference, but an approach that the mono folks have been following to bring up tests in new platforms is by disabling their build from libs.tests subset. https://github.com/dotnet/runtime/blob/220b01a58697eeae580d97cc47879217efd6628f/src/libraries/tests.proj#L23-L65

This way a local run of all libraries tests .ie: `build.cmd libs.tests -test -arch arm64 -os osx` would be clean. 

Anyway, not a strong preference, I'll leave it up to you since you are doing the bringup.

Reply:
Yeah I looked at that too.  I thought this way I could see the failures on my local builds if I didn't include IgnoreForCI.